### DISCUSSION TITLE
Translation

### DISCUSSION THREAD

**Discussion by akashramtohul4** [quasarframework/quasar#14856] [CLI - PWA mode]
Hello Friends,

Am looking for a better way to translate a page that contain **General Terms and Conditions** but the issue is that it 6 page.
I can use **i18n** key pair value but its too lengthy . can someone help me to find a better way 

example of Terms and Conditions
![example](https://user-images.githubusercontent.com/84211129/200768540-8d032e8a-ab05-44ec-87e1-624c769a38f1.png)

Thanks

**Comment by MilosPaunovic** [comment]
https://www.npmjs.com/search?q=translate

But, as this is not really Quasar related issue, try choosing one of the packages from the list above and, if necesarry, raise ticket on that repository.
